dallinger/prolific.py

```diff
@@ -505,7 +505,7 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
                 if endpoint == "/studies/":
                     # method="POST", endpoint="/studies/", json=payload
                     response = {
-                        "id": "study-id",
+                        "id": "66b0f8e34632badef5c8d1db",
```

Comment:
I think that '66b0f8e34632badef5c8d1db' should not be hardcoded but instead made a variable, this will help readability and maintainability

Reply:
Addressed